ArcGISRuntimeSDKQt_QMLSamples/Scenes/ChooseCameraController/ChooseCameraController.json

```diff
@@ -0,0 +1,5 @@
+{
+    "source" :  "ChooseCameraController.qml",
+    "otherSourceFiles" :  [],
+    "name" : "Choose Camera Controller"
```

Comment:
Note this JSON file is required to inform the sample viewer not just what files to display, but also what data to download!

Take a look at the Animate3D sample's json to see the missing bit!